tgui/packages/tgui/interfaces/common/BeakerDisplay.tsx

```diff
@@ -0,0 +1,67 @@
+import { BooleanLike } from 'common/react';
+import { AnimatedNumber, Box, Button, LabeledList } from '../../components';
+import { useBackend } from '../../backend';
+
+export type BeakerReagent = {
+  name: string;
+  volume: number;
+};
+
+export type Beaker = {
+  maxVolume: number;
+  transferAmounts: number[];
+  pH: number;
+  currentVolume: number;
+  contents: BeakerReagent[];
+};
+
+export type BeakerProps = {
```

Comment:
are we using these types elsewhere? if not, prefer not exporting them

Reply:
Yes we are. I did say this in the PR body 

> Created a common component for chem dispenser, portable chem dispenser & debug chem synthesizer to share in the UI code.

This is that component

Reply:
You're using the component and Beaker sure- I don't see any imports of BeakerReagent or BeakerProps

Reply:
Ah my bad it was like 1pm yesterday so I wasn't thinking straight. 
Yup I'll remove the export